setup.py

```diff
@@ -340,14 +340,22 @@ def find_tbb(tbb_root=None, tbb_include=None, tbb_link=None):
 modules = [m.replace(os.path.sep, '.') for m in modules]
 
 # Source files required for all modules.
-sources_in_all = []
+sources_in_all = [
+    os.path.join("cpp", "locality", "NeighborQuery.cc"),
+    os.path.join("cpp", "locality", "AABBQuery.cc"),
+    os.path.join("cpp", "locality", "LinkCell.cc"),
+    os.path.join("cpp", "locality", "NearestNeighbors.cc"),
+    os.path.join("cpp", "locality", "NeighborList.cc"),
+    os.path.join("cpp", "box", "Box.cc")
+]
```

Comment:
Do you actually need to add all of these back in? I would assume that at least LinkCell and NearestNeighbors could be removed, and I don't think Box.cc does anything (we should just remove it anyway).